src/pages/install.astro

```diff
@@ -0,0 +1,179 @@
+---
+import i18next, { t, changeLanguage } from "i18next"
+import Base from "../layouts/Base.astro"
+
+changeLanguage("en")
+
+const title = `Install | ${t("site.title")}`
+const description = t("site.description")
+---
+<!-- All changes (including styles) are limited to this page since it is temporary -->
+
+<Base title={title} description={description}>
+  <section class="section">
+    <div class="contentWrapperSm">
+      <img
+        alt="Web Monetization Icon"
+        class="wm-icon"
+        src="/img/wm-logo--animated.svg"
+        width="120"
+        height="86"
+      />
+      <h1 class="heading2">Web Monetization</h1>
+    </div>
+  </section>
+  <section>
+    <div class="contentWrapperLg">
+      <div class="cards twoColGrid">
+        <div class="card">
+          <div class="cardContent">
+            <h2 class="heading3">Support content with the web extension</h2>
+            <ul class="icons">
+              <li class="icon">
+                <a
+                  href="https://chromewebstore.google.com/detail/web-monetization/oiabcfomehhigdepbbclppomkhlknpii"
+                >
+                  <img
+                    alt="Chrome Web Store logo"
+                    src="/img/chrome-webstore.png"
+                  />
+                </a>
+              </li>
+              <li class="icon">
+                <a
+                  href="https://addons.mozilla.org/en-US/firefox/addon/web-monetization-extension"
+                >
+                  <img
+                    alt="Firefox Browser Add-ons"
+                    src="/img/Fx-Browser-icon.png"
+                  />
+                </a>
+              </li>
+              <li class="icon">
+                <a
+                  href="https://microsoftedge.microsoft.com/addons/detail/web-monetization/imjgemgmeoioefpmfefmffbboogighjl"
+                >
+                  <img alt="Edge Add-ons" src="/img/microsoft-edge.png" />
+                </a>
+              </li>
+            </ul>
+            <ul class="icons">
+              <li class="icon">
+                <a
+                  href="https://github.com/interledger/web-monetization-extension"
+                >
+                  <img alt="GitHub logo" src="/img/icon-github.svg" />
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <div class="card">
+          <div class="cardContent">
+            <h2 class="heading3">Get support with a Web Monetization link</h2>
+            <h2 class="heading3">
+                <a href="/link-tag/">Generate your link</a>
```

Comment:
"Generate your link" could be confusing, as if telling to generate payment pointer?

```suggestion
                <a href="/link-tag/">Generate your link tag</a>
```